tests/build_test.sh

```diff
@@ -238,6 +238,18 @@ function test_kernel_build_menu_cross_compilation_flags()
   assertEquals "($LINENO)" "$expected_result" "$output"
 }
 
+function test_kernel_build_clean() {
```

Comment:
See shfmt failure, and read: https://kworkflow.org/content/codingstyle.html#functions